cmake/Common/CompilerArmClangCommon.cmake

```diff
@@ -186,6 +186,14 @@ function(compiler_generate_binary_output TARGET)
 	add_custom_command(TARGET ${TARGET} POST_BUILD COMMAND ${CMAKE_ARMCCLANG_FROMELF} ARGS --bincombined --output=$<TARGET_FILE_DIR:${TARGET}>/${TARGET}.bin $<TARGET_FILE:${TARGET}>)
 endfunction()
 
+function(compiler_generate_hex_output TARGET)
+	add_custom_command(TARGET ${TARGET} POST_BUILD COMMAND ${CMAKE_ARMCCLANG_FROMELF} ARGS --i32combined --output=$<TARGET_FILE_DIR:${TARGET}>/${TARGET}.hex $<TARGET_FILE:${TARGET}>)
```

Comment:
I wonder if we could skip this, since we are going to be working with GCC only...

Reply:
I was thinking this could be upstreamed.